autogpt/cli.py

```diff
@@ -4,6 +4,7 @@
 
 @click.group(invoke_without_command=True)
 @click.option("-c", "--continuous", is_flag=True, help="Enable Continuous Mode")
+@click.option("--risk_avoidance", is_flag=True, help="Enable Risk Avoidance Mode")
```

Comment:
this is ugly, can it also work with `kebab-case`? (based on `--skip-reprompt` -> `skip_reprompt` probably yes)

Reply:
Of course, thats completely my bad. I'll change it right now.